go.mod

```diff
@@ -1,5 +1,5 @@
-//module github.com/jfrog/terraform-provider-xray
-module terraform-provider-xray
+module github.com/jfrog/terraform-provider-xray
+//module terraform-provider-xray
```

Comment:
??

Reply:
Removed, it was added to make provider work without git repo, when it was private